ansible_3par_docker_plugin/tasks/create_conf_file.yml

```diff
@@ -185,7 +185,7 @@
       path: /etc/hpedockerplugin/hpe.conf
       section: "{{ item[0] }}"
       option: "{{ item[1]['option'] }}"
-      value: "{{ 'https://' + INVENTORY[item[0]][item[1].value] + ':' + INVENTORY[item[0]][item[1].port_value] + '/api/v1' }}"
+      value: "{{ 'https://' + INVENTORY[item[0]][item[1].value] + ':' + INVENTORY[item[0]][item[1].port_value] | string + '/api/v1' }}"
```

Comment:
@bhagyashree-sarawate Why is "| string" needed here?

Reply:
If we do not give ,getting the following error
fatal: [10.50.128.110]: FAILED! => {"msg": "Unexpected templating type error occurred on ({{ 'https://' + INVENTORY[item[0]][item[1].value] + ':' + INVENTORY[item[0]][item[1].port_value] + '/api/v1' }}): coercing to Unicode: need string or buffer, int found"}

To fix the above error converted it into string for the string concatenation

Reply:
@bhagyashree-sarawate Either you can put the port in "<port number>" in property file or use "| string" when using the port number. It is because ansible treates 8080 as int when not given in quotes.